benchmark/bm_mutable_rotations.py

```diff
@@ -0,0 +1,49 @@
+"""Mutable sequence of rotations benchmark.
+The benchmark consists of a single mutable QNode evaluated several times,
+each evaluation having a larger number of rotations in the circuit.
+"""
+# pylint: disable=invalid-name
+import numpy as np
+
+import pennylane as qml
+
+n = 30
+n_min = 10
+n_max = 60
+n_step = 10
+
+n_wires = 1
+
+dev = qml.device("default.qubit", wires=n_wires)
+
+
+def circuit(a, b=1):
+    """Mutable quantum circuit."""
+    for idx in range(b):
+        qml.RY(a[idx], wires=[0])
+    return qml.expval(qml.PauliX(0))
+
+
+try:
+    circuit = qml.qnodes.QubitQNode(circuit, dev, mutable=True)
+except AttributeError:
+    circuit = qml.QNode(circuit, dev, cache=False)
```

Comment:
Is this to also be useful for older versions of PL?

Reply:
Yeah, to be able to benchmark versions before the "new-style" QNodes were introduced.